csi/node_server.go

```diff
@@ -136,8 +140,19 @@ func (ns *NodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 	}
 
 	if volumeCapability.GetBlock() != nil {
-		devicePath := volume.Controllers[0].Endpoint
+		devicePath := filepath.Join(stagingTargetPath, volumeID)
+		_, err := os.Stat(devicePath)
```

Comment:
Why use `stagingTargetPath` + `volumeID` instead of `stagingTargetPath` only? Shouldn't the stage path be still only for the volume?